extensions/wikidata/src/org/openrefine/wikidata/utils/EntityCache.java

```diff
@@ -77,6 +80,13 @@ public static EntityCache getEntityCache() {
         return _entityCache;
     }
 
+    public Stream<EntityDocument> getMultipleDocuments(List<EntityIdValue> entityIds) throws ExecutionException {
+        List<String> ids = new ArrayList<>();
+        for (EntityIdValue entityId : entityIds) {
+            ids.add(entityId.getId());
+        }
```

Comment:
You could probably use the streaming API for this instead: `entityIds.stream().map(…).collect(…)`